reviewkitteh/program.py

```diff
@@ -0,0 +1,62 @@
+"""Listen for GitHub pull requests and meow at random people.
+
+This program listens for GitHub pull request events and posts a message to a
+Slack channel when a pull request is opened or reopened. It then polls the
+pull request until it is closed or merged, updating the message with the
+current state of the pull request. Every 15 seconds, it also reads a random
+name from a Google Sheet and pages that person in the Slack channel.
+"""
+
+from datetime import datetime
+import itertools
+import os
+import time
+import random
+
+from autokitteh.github import github_client
+from autokitteh.google import google_sheets_client
+from autokitteh.slack import slack_client
+
+
+CHANNEL_ID = os.getenv("CHANNEL_ID")
+ORG_DOMAIN = os.getenv("ORG_DOMAIN")
+SHEET_ID = os.getenv("SHEET_ID")
+
+github = github_client("github_conn")
+googlesheets = google_sheets_client("googlesheets_conn").spreadsheets().values()
+slack = slack_client("slack_conn")
+
+
+def on_github_pull_request(event):
+    """Workflow's entry-point."""
+    pr = event.data.pull_request
+    msg = f"{pr.html_url} [{pr.state}]"
+    ts = slack.chat_postMessage(channel=CHANNEL_ID, text=msg)["ts"]
+
+    for i in itertools.count(start=1):
```

Comment:
I agree with that. My reasoning behind using the `itertools.count` is apparently it's more pythonic, at least according to the research I did. I'll change it back, but my thought was if we're targeting python users, this might be more familiar? On the other hand if we're targeting very basic python users, then `while` might be a friendlier option?